### ISSUE TITLE
Question: how to edit a freehand boundary that has been stored in an existing Activity

### ISSUE THREAD

**Issue by kitenetter** [BiologicalRecordsCentre/iRecord#367]
This applies to the Bringsty Common activity, group ID 374.

The activity was set up with a freehand boundary, but they wish to edit the boundary.

If I go into the activity edit page, the existing boundary is not shown - the filter map defaults to the whole of the UK.

Is it safe to just create and save a new freehand boundary in the activity - will this cause any complications for the records and users linked to that activity?

Would an alternative, and perhaps better, method be to create the new boundary in the My Sites area of the activity admin, and then apply the new boundary via the My Sites list in the activity filter settings?

**Comment by kitenetter** [comment]
In the end I did get the activity admin to set up a new boundary in her "My sites". I then edited the activity to use the "My sites" boundary instead of the previous freehand one.

The new boundary is now showing, but only a small proportion of the records that are within that boundary are being displayed. There should be dots over most of the site, not just round the edges:
https://www.brc.ac.uk/irecord/groups/explore?group_id=374&implicit=

What's the problem here?

**Comment by johnvanbreda** [comment]
You had drawn the shape of the site as a line, not a polygon (using the wrong tool). So, I did the following to fix it in the database to save you re-drawing (for reference):
1) Use st_astext(boundary_geom) to inspect the geometry.
2) In order to make a polygon, PostGIS needs the first and last point to be the same so it is complete. I used st_addpoint to update the boundary geom with the first point being added to the end.
3) I then used st_makepolygon to turn it into a polygon.
